activation/activation.go

```diff
@@ -223,6 +223,7 @@ func (b *Builder) PublishActivationTx(epoch types.EpochId) error {
 	atx := types.NewActivationTxWithChallenge(*b.challenge, activeIds, b.mesh.GetLatestView(), b.nipst, true)
 	activeSetSize, err := b.db.CalcActiveSetFromView(atx)
 	if epoch != 0 && (activeSetSize == 0 || activeSetSize != atx.ActiveSetSize) {
+		b.log.Error("empty active set size found! len(view): %d, view: %v", len(atx.View), atx.View)
```

Comment:
if you added this because Panic doesn't show the log message. @noamnelke fixes it in his PR. the `Panic` func calls `Fatal` and then `Panic` instead of logging the error and panicking.

Reply:
great!